worker/download.go

```diff
@@ -291,7 +290,7 @@ outer:
 		var resp *slabDownloadResponse
 		select {
 		case <-mgr.shutdownCtx.Done():
-			return errDownloadManagerStopped
+			return ErrShuttingDown
```

Comment:
Shouldn't this case be dropped altogether if we want the download to finish?

Reply:
that is true but this PR is not introducing graceful worker shutdown, it will be removed in #882